api/webservice/Portal/Users/Login.php

```diff
@@ -70,7 +81,8 @@ public function post()
 			'lastLogoutTime' => $row['logout_time'],
 			'language' => $row['language'],
 			'type' => $row['type'],
-			'CompanyId' => (\Api\Portal\Privilege::USER_PERMISSIONS !== $row['type']) ? $parentId : 0,
+			'companyId' => (\Api\Portal\Privilege::USER_PERMISSIONS !== $row['type']) ? $parentId : 0,
+			'companyDetails' => $companyDetails,
 			'logged' => true,
 			'checkStockLevels' => $checkStockLevels,
```

Comment:
You have duplicated data [check_stock_levels - checkStockLevels and $companyDetails['check_stock_levels']